app/(app)/feedback.tsx

```diff
@@ -1,20 +1,144 @@
-import React from 'react';
+import { AntDesign } from '@expo/vector-icons';
+import { useTheme } from '@react-navigation/native';
+import { useRouter } from 'expo-router';
+import React, { useState } from 'react';
+import { Pressable, StyleSheet, View } from 'react-native';
+import { TextInput } from 'react-native-gesture-handler';
+import FeedBackBanner from '../../assets/artworks/FeedBackBanner';
+import FeedBackRatingButton from '../../components/buttons/FeedBackRatingButton';
 import StyledText from '../../components/common/StyledText';
 import MainContainer from '../../components/container/MainContainer';
+import FeedbackSentModal from '../../components/modals/FeedbackSentModal';
+import { typography } from '../../config/typography';
 
-// TODO: implement feedback page
-/**
- * TASKS:
- * Implement feedback page
- * Should include a reactions component, a textarea and button
- */
+export type TypeRatingStates = {
+  icon: string;
+  text: string;
+  value: number;
+};
+
+const Feedback = () => {
+  const { colors } = useTheme();
+  const router = useRouter();
+  const [showModal, setShowModal] = useState(false);
+  const [_selectedRating, setSelectedRating] = useState(2);
+  const [description, setDescription] = useState('');
+  const ratingStates: Array<TypeRatingStates> = [
+    { icon: '😔', text: 'Bad', value: 0 },
+    { icon: '😐', text: 'Okay', value: 1 },
+    { icon: '😊', text: 'Great', value: 2 },
+  ];
 
-const feedback = () => {
   return (
-    <MainContainer preset="scroll">
-      <StyledText>feedback</StyledText>
+    <MainContainer preset="scroll" safeAreaEdges={['top']}>
+      <FeedbackSentModal showModal={showModal} />
+      <View style={styles.FeedBackBannerParent}>
+        <FeedBackBanner />
+        <Pressable style={styles.headerTitle} onPress={() => router.back()}>
+          <AntDesign name="arrowleft" size={24} color={'white'} />
+          <StyledText style={[{ color: colors.whiteConstant }]} size="lg">
+            Feedback
+          </StyledText>
+        </Pressable>
+      </View>
+      <View style={styles.FeedBackFormContainer}>
+        <StyledText size="lg" font="bold" variant="text" style={[styles.FeedBackFormTitle, { color: colors.primary }]}>
+          Your feedback helps us improve
+        </StyledText>
+        <View style={[styles.FeedBackForm, { backgroundColor: colors.background, borderColor: colors.border }]}>
+          <StyledText style={styles.FeedBackFormLabel} size="md">
+            How is/was the event
+          </StyledText>
```

Comment:
use size "base" as per the design

Reply:
Fixed in commit 487b35379151e9910c847f0eb80aed95840c7078